### PR TITLE
adding support for X-EconomicAppIdentifier header

### LINKED ISSUE #5: Add  X-EconomicAppIdentifier header
http://techtalk.e-conomic.com/e-conomic-soap-api-now-requires-you-to-specify-a-custom-x-economicappidentifier-header/?utm_campaign=2626709&utm_content=21060960743&utm_medium=email&utm_source=Emailvision

### PR DESCRIPTION
**Pull request by dinesh**
closes #5

### CODE REVIEW

File: lib/ruconomic/web_service.rb [hunk 1]
```diff
@@ -27,7 +32,7 @@ def call(document, action)
           c.max_redirects = Ruconomic.max_redirects
         end
       end
-
+
```
Comment (ta) [inline lib/ruconomic/web_service.rb]: Please avoid unnecessary white-space :)

**Comment by ta** [comment]
Hi @dinesh - thanks for your contribution. Could you add it to the configuration example in README?

**Comment by dinesh** [comment]
@ta corrected now.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ Ruconomic.configure do |config|
   config.agreement = "agreement"
   config.username = "username"
   config.password = "password"
+
+  ## change for your integration but try to keep the format same
+  configure.app_identifier = "MyCoolIntegration/1.1 (http://example.com/MyCoolIntegration/; MyCoolIntegration@example.com) Ruconomic/#{Ruconomic::VERSION}"
```
Comment (ta) [inline README.md]: You surely mean `config.app_identifier` ;-)

**Comment by ta** [comment]
@dinesh Had a single comment - please review.

**Comment by dinesh** [comment]
@ta Right, fixed the typo :)

**Comment by ta** [comment]
@dinesh Super - Thanks for your contribution!

**Pull request merged**